MSI/Day5/음양더하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(absolutes, signs) {
+    let total = 0;
+    for (let i = 0; i < absolutes.length; i++) {
+        if (signs[i]) {
+            total += absolutes[i];
+        } else {
+            total -= absolutes[i];
+        }
+    }
+    return total;
+}
```

Comment:
```suggestion
function solution(absolutes, signs) {
    let total = 0;
    
    for (let i = 0; i < absolutes.length; i++) {
        if (signs[i]) {
            total += absolutes[i];
        } else {
            total -= absolutes[i];
        }
    }
    return total;
}
```
변수 선언부와 for/if/while 문 등의 로직들은 줄바꿈으로 문맥을 분리함으로써 가독성을 키울 수 있습니다. 기억해주세요! 🤗

Reply:
확실히 가독성이 좋아졌네요 기억하겠습니다 감사합니다 !!